modules/hosted-control-planes-version-support.adoc

```diff
@@ -12,7 +12,20 @@ With each major, minor, or patch version release of {product-title}, two compone
 * HyperShift Operator
 * Command-line interface (CLI)
 
-The HyperShift Operator manages the lifecycle of hosted clusters that are represented by `HostedCluster` API resources. The HyperShift Operator is released with each {product-title} release.
+The HyperShift Operator manages the lifecycle of hosted clusters that are represented by `HostedCluster` API resources. The HyperShift Operator is released with each {product-title} release. After the HyperShift Operator is installed, it creates a `ConfigMap` called `supported-versions` in the HyperShift namespace. That `ConfigMap` describes the HostedCluster versions that can be deployed, as shown in this example:
```

Comment:
s/`ConfigMap`/`ConfigMap` object   --or-- config map

Should _as shown in this example_ be at the end of _it creates a `ConfigMap` called `supported-versions` in the HyperShift namespace_?

Reply:
I updated the text according to your feedback. Thanks!